easybuild/easyconfigs/l/LuaJIT2/LuaJIT2-v2.1-20220411-GCC-9.3.0.eb

```
@@ -0,0 +1,31 @@
+# This is the LuaJIT fork from openresty, not to be mixed up
+# with the original LuaJIT version!
+# Autor: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'ConfigureMake'
+
+name = "LuaJIT2"
```

Okay, let's indeed go for `LuaJIT-OpenResty` to make a clear distinction with the existing `LuaJIT`:

```suggestion
name = "LuaJIT-OpenResty"
```

---

That makes sense. I was not quite sure what was the best way forward here. Thanks!